libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextExplanationView/TextExplanationView.tsx

```diff
@@ -273,15 +249,14 @@ export class TextExplanationView extends React.PureComponent<
     }
   };
 
-  private switchQAPrediction = (): // _event?: React.FormEvent,
-  // _item?: IChoiceGroupOption
-  void => {
-    /*
-     * switch to the target predictions(starting or ending)
-     * TODO: add logic for switching explanation data
-     */
-    // if (item?.key !== undefined) {
-    //   this.setState({ qaRadio: item.key });
-    // }
+  private switchQAPrediction = (
+    _event?: React.FormEvent,
+    item?: IChoiceGroupOption
+  ): void => {
+    if (item?.key !== undefined) {
```

Comment:
```suggestion
    if (item?.key) {
```
Nit: curious if this may suffice

Reply:
yep that works